torchtune/_cli/ls.py

```diff
@@ -44,12 +44,12 @@ def main():
         examples:
             $ tune ls
             RECIPE                           CONFIG
-            full_finetune_distributed.py     full_finetune_distributed.yaml
-            lora_finetune_distributed.py     lora_finetune_distributed.yaml
+            full_finetune_distributed.py     llama2/7B_full, llama2/13B_full
+            lora_finetune_distributed.py     llama2/7B_lora, llama2/13B_lora
```

Comment:
Happy to finally see more than one config per recipe. Nit: maybe we can split the configs over separate lines? E.g.

```
RECIPE                                           CONFIG
full_finetune_distributed.py                     llama2/7B_full
                                                 llama2/13B_full
lora_finetune_distributed                        llama2/7B_lora
                                                 llama2/13B_lora
```

Reply:
Yep, my vision was to be over multiple lines and I think that's actually how it works right now?

Reply:
Yup, sorry I just need to update this example.